cocotb/log.py

```diff
@@ -69,6 +69,8 @@ def default_config():
     ``logging.basicConfig(..., force=True)`` (in Python 3.8 onwards), or by
     manually resetting the root logger instance, for which examples can be
     found online.
```

Comment:
```suggestion
    found online.
    
    An example of this can be found in the section on :ref:`rotating-logger`.
```

Reply:
Done (with the online examples part removed).